src/compile.rs

```diff
@@ -214,6 +218,22 @@ fn compile_target(
             ];
             rustc_args.extend(mac_args);
         }
+    } else if target.is_emscripten() {
+        shared_args.push("-Zbuild-std");
+        rust_flags
+            .get_or_insert_with(Default::default)
+            .push(" -C relocation-model=pic");
+        let emscripten_args = &[
+            "-C",
+            "relocation-model=pic",
+            "-C",
+            "target-feature=+mutable-globals",
```

Comment:
You probably only need to pass `relocation-model=pic` once and I think `-Ctarget-feature=+mutable-globals` isn't needed anymore.
```suggestion
        let emscripten_args = &[
            "-C",
            "relocation-model=pic",
```